### DISCUSSION TITLE
Backend Host Configuration

### DISCUSSION THREAD

**Discussion by lanefu** [fosshostorg/aarch64#23] [General]
What file system and storage config are being using for the running VMs?

Mostly wanted to give an FYI in case using BTRFS in /opt..

if using qcow2 images.. you want to avoid them residing on a COW filesystem.... that's a BTRFS default.... BUT you can mount   it with `nodatacow` option.. or use a cli command to apply to specific folders or files.

**Comment by natesales** [comment]
We're using ext4 for the host filesystem. The hosts only have a single SSD each so it didn't make much sense to use anything more complex than ext4. Of course a single drive is **very far** from ideal, but it's the hardware that we have to work with at the moment so we'll have to make it work. In the future when we get more drives we'll probably switch to ZFS similar to how the x86 Fosshost hypervisors are configured. /opt/ is actually in the root partition, I chose to keep VM data in there to make the transition easier when we end up going with a redundant storage solution.

**Comment by lanefu** [comment]
K probably best optimizations that can be done for now are probably just assure the ext4 is mounted with the `noatime` flag.

I was looking into ioscheduler options for kernel 4.19.  If tenancy is low sticking with none is probably best.  If the box is heavily oversubscribed may want to look into BFQ
